lib/cpu.js

```diff
@@ -19,7 +20,7 @@ var getUsage = function(cb) {
 
 var checkInterval = {};
 
-module.exports = function(influxclient) {
+module.exports = function(metricsClient) {
```

Comment:
This says `metricsClient`, indicating a single one, where `lib/counter` above says `metricsClients` indicating potentially more than one. Should they be more similar, or are they intentionally different like this?

~~Edit: looking at `lib/guage.js` below, I think `lib/counter.js` above should be `metricsClient`, does that sound right?~~

Edit2: I now thing this should be `metricsClients`, as it should be in other places like `lib/guage.js`.